src/array/ops/cast.rs

```diff
@@ -260,23 +282,57 @@ fn extract_python_to_vec<
         dtype if dtype.is_integer() => Ok("int"),
         dtype if dtype.is_floating() => Ok("float"),
         dtype => Err(DaftError::ValueError(format!(
-            "We only support numeric types when converting to FixedSizeList, got {dtype}"
+            "We only support numeric types when converting to List or FixedSizeList, got {dtype}"
         ))),
     }?;
 
     let py_type_fn = { PyModule::import(py, pyo3::intern!(py, "builtins"))?.getattr(pytype)? };
+    let py_memory_view = py
+        .import("builtins")?
+        .getattr(pyo3::intern!(py, "memoryview"))?;
+    let np_as_array_fn = py.import("numpy")?.getattr(pyo3::intern!(py, "asarray"))?;
```

Comment:
Can we also guard these imports since numpy isn't a required dep for us?

Reply:
Oh I didn't know that, that's good to know!